app/views/internal/configs/show.html.erb

```diff
@@ -245,6 +245,31 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <%= render partial: "card_header",
+                     locals: {
+                       header: "Jobs",
+                       state: "collapse",
+                       target: "jobsBodyContainer",
+                       expanded: "false"
+                     } %>
+          <div id="jobsBodyContainer" class="card-body collapse hide" aria-labelledby="jobsBodyContainer">
+            <div class="form-group">
+              <%= f.label :jobs_url %>
+              <%= f.text_field :jobs_url,
+                               class: "form-control",
+                               value: SiteConfig.jobs_url,
+                               placeholder: "Jobs URL" %>
+              <div class="alert alert-info">URL of the website where open positions are posted.</div>
+
+            <div class="form-group">
+              <%= f.label :display_jobs_before_search %>
+              <%= f.check_box :display_jobs_before_search, checked: SiteConfig.display_jobs_before_search %>
+              <div class="alert alert-info">Display a small jobs banner that points users to the jobs url when they search job or jobs in the search box</div>
```

Comment:
```suggestion
              <div class="alert alert-info">Display a jobs banner that points users to the <%= community_name %> jobs page when they type "job" or "jobs" in the search box</div>
```

Reply:
AH thank you! I was struggling to figure out what the helper was for that!